content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Open sourcing Workshops-on-Demand - Part 5: Create a new workshop"
+date: 2025-09-26T08:24:03.351Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - Workshops-on-Demand
+  - Open Source
+---
+I﻿n this new article that is part of our series dedicated on [open sourcing of our Workshops-on-Demand project](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/), I will focus on the steps necessary  to build up a new workshop. I already covered most of the infrastructure parts that support the workshops. In my previous posts, I covered setting up the infrastructure to support the workshops. Now let's focus a little more on the content creation.
```

Comment:
Remove the word "new" and just say "In this article that is part..."

Recommend combining the 2nd and 3rd sentence like this:

"In my previous posts, I have already covered most of the pieces on how to set up the infrastructure to support the workshops."